### DISCUSSION TITLE
Still waiting for the Kubernetes API (OKD on OpenStack)

### DISCUSSION THREAD

**Discussion by hasandjamil** [okd-project/okd#1187] [Q&A]
Hello everyone, kindly help me to let me know and give me an insight to this problem.
I am expecting some help from all of the experts here, thank you.

This is perspective from worker console.
![Screenshot from 2022-04-19 16-38-47](https://user-images.githubusercontent.com/22232023/163975803-b808598b-7daa-43a0-be55-ea7959a1d5eb.png)

Below is the error when I executed create cluster command.
![Screenshot from 2022-04-19 16-41-09](https://user-images.githubusercontent.com/22232023/163976297-b761a985-79c5-4667-bc72-0136cff9ee72.png)

**Comment by fcxone** [comment]
I've same problem with you, hope someone can help us.

**Comment by JaimeMagiera** [comment]
"I am expecting some help from all of the experts here, thank you."

As a volunteer-based community distribution, we do our best to help folks. However, there are no guarantees.

Could  you please let the community know what version of OKD and on what platform you are attempting to install? Also, could you follow the instructions here for gathering installation logs for community members to see?

https://docs.okd.io/latest/support/troubleshooting/troubleshooting-installations.html#gathering-bootstrap-diagnostic-data_troubleshooting-installations
